content.md

```diff
@@ -705,8 +705,12 @@ cards. When this template is used, only one Y can be done per X with
 this card. For instance, you may move blood from your vampire to your
 pool with Vessel only once per unlock phase.
 
-**"Lock X to do Y":** These effects require an unlocked minion to be
-used, they cannot be used by locked minions.
+**"Cost X to do Y":** The "cost X to do Y" template is used on several
+cards. A minion or Methuselah playing a card or using an effect with
+this template will have to pay cost X in order to play the card or use
+the effect Y. For example, an effect with "lock a location to lock
+a minion" require the Methuselah to lock an unlocked location they
+control to lock a minion.
```

Comment:
The proposal could use a reasoning why the new content is better.

I find the previous content clearer for simple "lock x to y" cases. Using "cost" is confusing because "cost" is usually used for blood/pool cost, not in the MtG-like `cost: effect` sense this proposal seems to emulate.

How many effects we have where "cost x to do y" is anything else than "lock" or "burn a blood"? Burning blood does not seem to confuse people, locking does (evidence: rulings).

Reply:
I agree. "Lock to" is the only case where questions are asked. "Burn X blood to...", "Discard X cards to...", "Burn the Edge to..." are not an issue for that matter (they can be for other matters, but those matters are not covered here anyway).